myapp/spec/requests/admin/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin::Users', type: :request do
+  describe 'DELETE /admin/users/:id' do
```

Comment:
The current test only covers the case of trying to remove the last administrator, but it would be even better if it also covered the scenario of changing the administrator's permissions!

Reply:
thank you ！I added some cases